packages/dapi-grpc/src/platform/proto/org.dash.platform.dapi.v0.rs

```diff
@@ -743,12 +743,15 @@ pub mod get_data_contract_history_response {
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct GetDocumentsRequest {
     #[prost(bytes = "vec", tag = "1")]
+    #[serde(with = "crate::deserialization::base64string")]
     pub data_contract_id: ::prost::alloc::vec::Vec<u8>,
     #[prost(string, tag = "2")]
     pub document_type: ::prost::alloc::string::String,
     #[prost(bytes = "vec", tag = "3")]
+    #[serde(with = "crate::deserialization::base64string")]
```

Comment:
I don't understand why

Reply:
It is needed to parse test vectors in `rs-drive-proof-verifier` tests.
Tools like postman or grpcui display base64. Without this, serde_json errors with `Error("invalid type: string \"B1cq2CW1ArO6OkMr6qLH8TRjw+JRQs0RTOqfjgeBpl8=\", expected a sequence`.

If we don't provide this conversion, serde_json encodes it as sth like `[123,124,125]` . It also decodes base64 incorrectly. 

It's ugly and generates additional task to convert data between formats before putting it into test vector.

However, I found a way to do it in a much better way in commit 89577f5ab878df54c0f4a842e6420cbca9f4f4ed , so I removed majority of these annotations.